opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml

```diff
@@ -268,6 +268,7 @@
          <parameter key="retry" value="2"/>
          <parameter key="timeout" value="3000"/>
          <parameter key="ignoreStandBy" value="false"/>
+         <parameter key="reportAlarms" value=""/>
```

Comment:
Is it required to force a merge for updating users with an empty new introduced parameter?

Reply:
no, empty string is the default

Reply:
You could avoid touching the config-tester, by making reportAlarms="" when the whole parameter is missing and make it backward compatible.

Reply:
I think I understood you correctly, the default value "" is also applied in code even the reportAlarms entry does not exist. Of course we can also remove the line here if you feel better :-)